src/main/webapp/app/quiz/participate/quiz.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnDestroy, OnInit, ViewChildren, QueryList } from '@angular/core';
```

Comment:
Correct me if I am wrong, but all changes in this file are unnecessary?

Reply:
you are right - I reverted